packages/service-library/src/servicelib/aiohttp/application_setup.py

```diff
@@ -62,12 +63,13 @@ class SetupMetadataDict(TypedDict):
 
 def _parse_and_validate_arguments(
     module_name, depends, config_section, config_enabled
-) -> Tuple:
+) -> tuple:
```

Comment:
could you specify the types returned in this tuple as well?

Reply:
but they are the same as the input arguments ... :-)